test/e2e/json-rpc/eth_accounts.spec.js

```diff
@@ -0,0 +1,48 @@
+const { strict: assert } = require('assert');
+const { convertToHexValue, withFixtures } = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('eth_accounts', function () {
+  const ganacheOptions = {
```

Comment:
could we use the `defaultGanacheOptions` instead? given they are the same, we could then skip this piece of code.
Example: https://github.com/MetaMask/metamask-extension/blob/develop/test/e2e/tests/eth-sign.spec.js#L20

Reply:
Done in [3f4323d](https://github.com/MetaMask/metamask-extension/pull/20963/commits/3f4323d5abc0b8c1993d5bcf98015f12a5405d5e)